misc_scripts/change_toolchain_version.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python3
+"""
+change_toolchain_version: change the toolchain version in the GitHub Actions
+workflow configurations.
+"""
+
+# isort: STDLIB
+import argparse
+
+KEY_LSRT = r"# LOWEST SUPPORTED RUST TOOLCHAIN"
+KEY_CDRT = r"# CURRENT DEVELOPMENT RUST TOOLCHAIN"
+TOOLCHAIN_CHOICES = ["lowest", "current"]
+
+
+def search_file(args, search_key, old_verstring, new_verstring, filename):
+    """
+    Read the file.
+    """
+    with open(filename, "r") as file:
+        for line in file:
+
+            if search_key in line:
+                templine = line.replace(old_verstring, new_verstring)
+                if args.new_version not in templine:
+                    raise ValueError("Old version not in file")
```

Comment:
Two requests for this line:
1. Please change ```ValueError``` to ```RuntimeError```. ```ValueError``` is seriously abused by Python programmers, really only the interpreter should use it.
2. Make the error message a bit more expressive of what's really going on.

Reply:
OK; I initially had the impression that the error was to express something wrong with the value of the variable.  (Maybe that's the origin of programmers incorrectly using it.)

As for the error message, I'm concerned that I might have to change the flow of this, once we change the program to write to a file.  The intent I had in mind was that, if the `replace()` call fails to properly add the new version, then it's most likely because the old version wasn't there.  That was why I went with, "Old version not in file".

So I'm not sure if I want to change the message yet, just in case we have to change the error flow here.  I'm just happy that the program seems to properly prevent incorrect version changes right now.